test/client/api.js

```diff
@@ -344,9 +344,13 @@ module.exports = config => {
     return tokens.SessionToken.fromHex(sessionTokenHex)
       .then(
         function (token) {
+          let url = this.baseURL + '/certificate/sign'
+          if (options.service) {
+            url += '?service=' + options.service
```

Comment:
No biggie but for future reference, I think we tend to prefer ES6 template strings for new code, instead of concatenating with `+`.